Gemfile

```diff
@@ -5,3 +5,10 @@ group :deployment do
   gem 'package_cloud'
   gem 'rake'
 end
+
+group :development do
+  # Newer versions don't work with Ruby 2.5 and 2.6
+  gem 'domain_name', '~> 0.5.20190701'
+  # Newer versions don't work with Ruby 2.5
+  gem 'thor', '~> 1.2.2'
```

Comment:
We could get rid of these if we drop support for anything older than Ruby 3.1 which is the oldest supported Ruby release